projects/rero/ng-core/src/lib/record/autocomplete/autocomplete.component.ts

```diff
@@ -114,12 +116,17 @@ export class AutocompleteComponent implements OnInit {
           };
         }
       }
-      this.dataSource = new Observable((observer: any) => {
+      this.dataSource = new Observable((suggestions$: Subscriber<{}>) => {
```

Comment:
I read somewhere that `object` (lowercase) is better than `{}` or `any` or `Object`.

* `object` is more restrictive as you can't assign primitive types.
* `any` accepts all input values without compilation checks.
* With`{}` (which is equivalent to `Object`) you can assign a primitive type.

Reply:
Good to know. Thanks.